### PR TITLE
Fix reference to User -> UserEntity

### PR DESCRIPTION
**Pull request by cyamonide**
Data class `User` was replaced with `UserEntity` in #13, missed in #14

### CODE REVIEW

**Comment by cyamonide** [comment]
Might we want to rename this data class to `MatchEntity` (following the convention set by `UserEntity`)?

**Comment by SPahooja** [comment]
It should be user only, dont worry about this I'm still in the process of working it out. I had to merge that to provide the domain package to decoupel the UI needs

File: app/src/main/java/com/uwcs446/socialsports/services/match/Match.kt [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 import com.google.type.DateTime
-import com.uwcs446.socialsports.services.user.User
+import com.uwcs446.socialsports.services.user.UserEntity
 
 data class Match(
     val id: String,
     val type: MatchType,
     val title: String,
     val description: String,
     val time: DateTime,
```
Comment (akassab) [inline app/src/main/java/com/uwcs446/socialsports/services/match/Match.kt:11]: I'm suggesting another field "createdAt" with type  `DateTime`.

Comment (akassab) [reply app/src/main/java/com/uwcs446/socialsports/services/match/Match.kt:11]: ^ You can choose to ignore this since the PR was just for a fix

**Pull request closed without merging**